robot/control/Src/motion-control/VelocityKalmanTuner/KalmanFilter.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import scipy.linalg
+
+
+class KalmanFilter(object):
+    def __init__(self, x_hat_init, init_covariance, steady_state=False):
+        """
+        Initialize new Kalman Filter Object with
+        (3 x 1) state : [v_x, v_y, omega] ([m/s, m/s, rad/s])
+        (4 x 1) control input: [motor_1, motor_2, motor_3, motor_4] (% V)
+        (5 x 1) output: [enc_1, enc_2, enc_3, enc_4, gyro_vel] (rad/s)
+
+        x_hat: (3 x 1) state vector
+        A: (3 x 3) matrix: transition/dynamics matrix
+        B: (4 x 4) matrix: control matrix
+        P: (3 x 3) matrix: covariance
+        H: (5 x 3) matrix
+        Q: (3 x 3) matrix
+        R: (5 x 5) matrix:
+        """
+        self.step_response = False
+        self.steady_state = steady_state
+
+        # Deg to Rad
+        self.wheel_angles = np.deg2rad([180 - 30,
+                                        180 + 39,
+                                        360 - 39,
+                                        0 + 30])
+        self.wheel_dist = 0.0798576
+        self.wheel_radius = 0.02786
```

Comment:
Probably add a comment indicating where these numbers are from (so its easy to change later)